doc/overview.txt

```diff
@@ -49,19 +49,19 @@ is a valid instruction sequence:
     sljit_emit_op1(compiler, SLJIT_MOV32,
         SLJIT_R0, 0, SLJIT_MEM1(SLJIT_R1), 0);
     // An int32_t value is loaded into SLJIT_R0
-    sljit_emit_op1(compiler, SLJIT_NEG32,
```

Comment:
neg is subtract from immediate 0. now all optimizations are moved to subtract, so neg is redundant